src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs

```diff
@@ -100,9 +102,11 @@ public void Configure(string name, OpenIdConnectOptions options)
         private async Task<OpenIdClientSettings> GetClientSettingsAsync()
         {
             var settings = await _clientService.GetSettingsAsync();
-            if ((await _clientService.ValidateSettingsAsync(settings)).Any(result => result != ValidationResult.Success))
+            var errorResult = (await _clientService.ValidateSettingsAsync(settings)).Where(r => r != ValidationResult.Success);
+
+            if (errorResult.Any())
```

Comment:
```suggestion
            if (errors.Any() && _logger.IsEnabled(LogLevel.Warning))
```

Reply:
if the `LogLevel.Warning` is not enabled , It will return an error setting information